numba/tests/test_np_functions.py

```diff
@@ -2577,6 +2585,80 @@ def test_logspace3_exception(self):
         self.assertIn('The third argument "num" must be an integer',
                       str(raises.exception))
 
+    def test_geomspace(self):
+
+        def inputs():
+            #start, stop
```

Comment:
I think some tests for negative numbers would be good. Might also be worth updating the implementation to reflect NumPy's implementation, this might help with handling of complex numbers? xref: https://github.com/numpy/numpy/blob/ae13307732b430daa56c6a85ab56006611a5b6ba/numpy/core/function_base.py#L287

Reply:
Thx, this fixed the issue.